starboard/android/apk/app/src/main/java/dev/cobalt/media/VideoFrameReleaseTimeHelper.java

```diff
@@ -118,7 +120,17 @@ public void disable() {
    */
   @SuppressWarnings("unused")
   @UsedByNative
-  public long adjustReleaseTime(long framePresentationTimeUs, long unadjustedReleaseTimeNs) {
+  public long adjustReleaseTime(
+      long framePresentationTimeUs, long unadjustedReleaseTimeNs, double playbackRate) {
+    if (playbackRate == 0) {
+      return unadjustedReleaseTimeNs;
```

Comment:
If `playbackRate` is 0, shall we still release the frame?  If not, is there an option in the VideoRenderAlgorithm to retry the releasing of the frame when `playbackRate` is changed to non-zero?
Note that we may always want to release a frame if we have never released one, i.e. at the playback startup, even if the playback rate is 0.

Reply:
When the video is prerolling or paused, the rate could be  0 and we still want to release the frame in these cases. So, to avoid complicating the logic too much, I suggest to keep it as is.